### PR TITLE
Add in repository quality measures.

### PR DESCRIPTION
**Pull request by garrypolley**
* Added TravisCI configuration
* Added CodeClimate Badge

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+   Python: false
+   PHP: false
+#
+# ---Exclude Files or Directories---
+# List the files or directories you would like excluded from our analysis.
+# For help setting your exclude paths:
+# http://docs.codeclimate.com/article/166-excluding-files-folders
+#
+exclude_paths:
+ - 'build/*'
```
Comment (sambao21) [inline .codeclimate.yml]: code climate still is looking at this and gave you an 'F' :fearful: are you sure this is working?

**Comment by sambao21** [comment]
FYI, saw this in the travis build 

```
npm WARN package.json canadarm@1.0.0 No license field.
```

**Comment by garrypolley** [comment]
Code Climate will only work after it is merged to master. At least from what I can see.

File: .codeclimate.yml (hunk 1 above)
Comment (garrypolley) [inline .codeclimate.yml]: Code Climate will only work after it is merged to master. At least from what I can see.

**Comment by gneatgeek** [comment]
:+1:

**Comment by garrypolley** [comment]
@sambao21  If you're good with this I'm going to merge it in the next few days.

**Comment by sambao21** [comment]
:+1:

**Pull request closed without merging**